xtable-utilities/src/main/java/org/apache/xtable/utilities/RunSync.java

```diff
@@ -142,63 +221,58 @@ public static void main(String[] args) throws IOException {
     ConversionSourceProvider<?> conversionSourceProvider =
         ReflectionUtils.createInstanceOfClass(sourceProviderClass);
     conversionSourceProvider.init(hadoopConf);
+    return conversionSourceProvider;
+  }
 
+  public static List<String> getTableFormatList(DatasetConfig datasetConfig) throws IOException {
+    // Retrieve table format list
     List<String> tableFormatList = datasetConfig.getTargetFormats();
```

Comment:
Why do we need a method that is wrapping a getter?

Reply:
Changes have been done.